### PR TITLE
added page inventor/sealed

### PR DESCRIPTION
**Pull request by shishir0019**

### CODE REVIEW

**Comment by shishir0019** [comment]
Screenshot 2022-05-13 at 1 56 09 AM

**Review by ardeay — CHANGES_REQUESTED** [review]
Please change the top level directory of this file to `inventory`, so its `pages/inventory/sealed.vue`

**Comment by ardeay** [comment]
The global search component is here https://github.com/EchoMTG/website/blob/main/components/GlobalSearch.vue

**Comment by ardeay** [comment]
You can see an example of global search being imported here https://github.com/EchoMTG/website/blob/a09029bc01e1063b60e70fe54842ab7a89e2f6d7/components/NavBar.vue#L152

File: pages/inventor/sealed.vue [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                            <a v-bind:href="item.echo_url">{{item.name}}</a>
+                        </td>
+                        <td>{{symbol}} {{item.tcg_mid}}</td>
+                        <td>
+                            <span class="percentage red down" v-if="item.gain < 0">{{item.gain}}%</span>
+                            <span class="percentage green up" v-if="item.gain > 0">{{item.gain}}%</span>     
+                        </td>
+                        <td>{{symbol}} <input class="adjust-box" data-call="inventory/adjust/" v-bind:value="item.price_acquired"/></td>
+                        <td>
+                            <input class="adjust-box input is-small acquired-date-input" type="date"  data-call="inventory/adjust_date/" v-bind:value="item.date_acquired_html"/>
```
[the remark this answers is not in the document]
Comment (ardeay) [reply pages/inventor/sealed.vue:77]:
```$(document).on("change",".adjust-box",function(e){
		var url = "/api/"+$(this).attr('data-call')+"id="+$(this).closest('tr,li,div,article').attr('data-id')+'&value='+$(this).val().replace(/\//g,'-');
		$.ajax(url);
	});```

[the remark this answers is not in the document]
Comment (ardeay) [reply pages/inventor/sealed.vue:77]: have it call a new function on the component that hits this endpoint

`/api/inventory/adjust_date/?id=[row_id]&value=[date]`

**Review by ardeay — CHANGES_REQUESTED** [review]
image

File: pages/inventory/sealed.vue [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    },
+    updateStatus: function updateStatus() {
+      this.status++
+    },
+    updatePrice(input, item) {
+      let token = this.$cookies.get('token');
+      let id = item.inventory_id;
+      let value = input.target.value;
+
+      let url = `${process.env.API_DOMAIN}/inventory/adjust/id=${id}&adjusted_price=${value}&auth=${token}`;
```
Comment (ardeay) [inline pages/inventory/sealed.vue]: Remove the first forward slash from the string `/inventory/....` because the request is making a bad url 
image

**Review by ardeay — APPROVED** [review]

**Pull request merged**